app/views/schools/observations/index.html.erb

```diff
@@ -1,29 +1,59 @@
 <h1>Temperature recordings</h1>
 <h2><%= @school.name %></h2>
 
-<% if @school.observations.any? %>
-  <table class="table table-striped">
+<% if @observations.any? %>
+  <table class="table table-striped table-sm">
     <thead>
       <tr>
-        <th>At</th>
-        <th>Observation</th>
-        <th>Recording location</th>
-        <th>Temperature &deg;C</th>
+        <th>Recording date</th>
+        <% @locations.each do |location| %>
+          <th><%= location.name %></th>
+        <% end %>
+        <% if can? :delete, Observation %>
+          <th>Actions</th>
+        <% end %>
       </tr>
     </thead>
     <tbody>
-      <% @observations.each do |observation| %>
-        <% observation.temperature_recordings.each_with_index do |temperature_recording, index| %>
-          <tr scope="row">
-            <td><%= nice_date_times observation.at %></td>
-            <td><%= observation.description %></td>
-            <td><%= temperature_recording.location.name %></td>
-            <td><%= temperature_recording.centigrade %></td>
-          </tr>
-        <% end %>
+      <% @observations.order(at: :desc).each do |observation| %>
```

Comment:
There's a simple pivot going on here, it does the trick for now, but if you've a more elegant way of doing it, then I'm all ears @jamesalmond !